src/nvim/eval.c

```diff
@@ -11493,20 +11493,20 @@ static void f_islocked(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 ///                 a dictionary, will give an error if not.
 /// @param[out]  rettv  Location where result will be saved.
 /// @param[in]  what  What to save in rettv.
-static void dict_list(typval_T *const tv, typval_T *const rettv,
+static void dict_list(typval_T *const tv_, typval_T *const rettv,
```

Comment:
Why not rename the loop variable to `tv_item` or something? In general it seems weird to corrupt a public name for the sake of not renaming a more private name.